### DISCUSSION TITLE
Terminal half life of midazolam in PBPK model

### DISCUSSION THREAD

**Discussion by MarjoleinvB** [Open-Systems-Pharmacology/Forum#1488] [Questions & Problems]
Hi Everyone,

The terminal half life of midazolam is very short. Due to how PK Sim calculates the half life, I get wrong half lifes (very long ~30h) out of my simulations. 

This is how PK-Sim calculates the half life: Terminal half life time (calculated from the terminal 10% of data points). 
I tried to change the sampling data points and the time till sampling, but this does not work very well. 

What is the right way to simulate compounds/drugs with a very short half life to get the right AUC_inf, Cmax and T1/2? 

Thank you!

Marjolein

**Comment by JohannaZH2** [comment]
Hi Marjolein, 
I read you question and would like to ask you whether you could find a way how to influence half-life value in PK Sim Simulations? I would be very happy to discuss this topic with you. 
We try to get used to PK Sim and it is a common problem with our simulations, that half-life is too long as it should be. 
We tried to chance logP and partition coefficient and cellular permeability, but it was not very successful. 
Thank you and I am looking forward to hearing from you. 
Best, Johanna

**Reply by StephanSchaller** [reply]
Hi Johanna,

Have you tried running a sensitivity analysis for t1/2 to find the most influential parameters? That's what you get for the "Chung 2006" PO simulation in the template model:
![image](https://github.com/Open-Systems-Pharmacology/Forum/assets/25477643/4ba98a9e-e925-4b09-9577-b28b8770c4aa)
Here half-life is 2.4 h.

What type of simulation are you running, IV or PO? How long are you simulating (long enough to actually get into terminal clearance (> 6 hours))?
Half-life sensitivity might depend on what clearance processes are implemented. For Midazolam, we have liver and kidney, so you might want to look at Vmax for CYP3A4 or parameters for glomerular filtration.

Best,
Stephan

**Comment by MarjoleinvB** [comment]
Hi Johanna,

Thank you for contacting me! I am afraid I have to disappoint you. Due to lack of time, I temporarily stopped using and learning PK Sim and switched to Simcyp. (My research group has more experience with that platform).
However, I can advise you to put your problem on the Github forum with specifics on which model you use. I really quickly got a response by one of the PK Sim experts.

My question was focussed on that I want to use t1/2 as PK parameter to validate my PBPK models (together with Cmax and AUC). They only responded they don't advise t1/2, since it is calculated by the last 10% of data points of the terminal slope. With this in mind, the problem of the long half-life might be caused if the drug is not completely eliminated in the simulation or another factor why the last part of the curve is not correctly.

I am really interested in the solution. I hope to pick up my work in PK Sim in a couple of months and will ask a colleague who recently started to use PK Sim if he has problems with t1/2.  If I figure out the problem, I am happy to discuss this with you. I hope you will find the solution.

Best,
Marjolein

Van: JohannaZH2 ***@***.***>
Verzonden: vrijdag 29 september 2023 12:06
Aan: Open-Systems-Pharmacology/Forum ***@***.***>
CC: Borselen, Marjolein van ***@***.***>; State change ***@***.***>
Onderwerp: Re: [Open-Systems-Pharmacology/Forum] Terminal half life of midazolam in PBPK model (Discussion #1488)


Hi Marjolein,
I read you question and would like to ask you whether you could find a way how to influence half-life value in PK Sim Simulations? I would be very happy to discuss this topic with you.
We try to get used to PK Sim and it is a common problem with our simulations, that half-life is too long as it should be.
We tried to chance logP and partition coefficient and cellular permeability, but it was not very successful.
Thank you and I am looking forward to hearing from you.
Best, Johanna

-
Reply to this email directly, view it on GitHub<https://github.com/Open-Systems-Pharmacology/Forum/discussions/1488#discussioncomment-7143040>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A6ZRZGZVYB4RG372CTOFV4TX42MSBANCNFSM6AAAAAAWKVGWLE>.
You are receiving this because you modified the open/close state.Message ID: ***@***.******@***.***>>

De informatie in dit bericht is uitsluitend bestemd voor de geadresseerde. Aan dit bericht en de bijlagen kunnen geen rechten worden ontleend. Heeft u deze e-mail onbedoeld ontvangen? Dan verzoeken wij u het te vernietigen en de afzender te informeren. Openbaar maken, kopi?ren en verspreiden van deze e-mail of informatie uit deze e-mail is alleen toegestaan met voorafgaande schriftelijke toestemming van de afzender. Het Radboudumc staat geregistreerd bij de Kamer van Koophandel in het handelsregister onder nummer 80262783.

The content of this message is intended solely for the addressee. No rights can be derived from this message or its attachments. If you are not the intended recipient, we kindly request you to delete the message and inform the sender. It is strictly prohibited to disclose, copy or distribute this email or the information inside it, without a written consent from the sender. Radboud university medical center is registered with the Dutch Chamber of Commerce trade register with number 80262783.

**Comment by JohannaZH2** [comment]
Hi Stephan,

thank you very much for you reply.
That`s a great idea to run a sensitivity analysis. We haven’t done it yet.
I would like to have one more question about your diagram. Am I interpreting correctly that the parameter that deviates the most from zero has the highest influence on in our example, the half-life?

We simulate cytarabine given IV for 3 hours. Cytarabine has very shot half-life, initial only 10 minutes and our simulation has more than 60 hours:
***@***.***

In our simulation, we changed LogP so that Cmax matched our target. As the next step, we changed the hepatic clearance. That helped to reduce the half-life and to improve the AUC matching the target, but our Cmax was changed too and did not matched the target any more.
Do you know  the reason for such behavior of PK-Sim?

Thank you very much for you help!

Best, Johanna

Von: esqlabs GmbH ***@***.***>
Gesendet: Montag, 2. Oktober 2023 13:41
An: Open-Systems-Pharmacology/Forum ***@***.***>
Cc: Bouzid Johanna ***@***.***>; Comment ***@***.***>
Betreff: Re: [Open-Systems-Pharmacology/Forum] Terminal half life of midazolam in PBPK model (Discussion #1488)


Hi Johanna,

Have you tried running a sensitivity analysis for t1/2 to find the most influential parameters? That's what you get for the "Chung 2006" PO simulation in the template model:
[image]<https://user-images.githubusercontent.com/25477643/271958780-4ba98a9e-e925-4b09-9577-b28b8770c4aa.png>
Here half-life is 2.4 h.

What type of simulation are you running, IV or PO? How long are you simulating (long enough to actually get into terminal clearance (> 6 hours))?
Half-life sensitivity might depend on what clearance processes are implemented. For Midazolam, we have liver and kidney, so you might want to look at Vmax for CYP3A4 or parameters for glomerular filtration.

Best,
Stephan

—
Reply to this email directly, view it on GitHub<https://smex-ctp.trendmicro.com:443/wis/clicktime/v1/query?url=https%3a%2f%2fgithub.com%2fOpen%2dSystems%2dPharmacology%2fForum%2fdiscussions%2f1488%23discussioncomment%2d7164119&umid=348ab537-dc66-4575-9714-76943461be42&auth=f075895abcc5d3a93325341b3e31d07f62e0e5ed-8c26ecaa11c962ab144412c05a35b9df0ef75524>, or unsubscribe<https://smex-ctp.trendmicro.com:443/wis/clicktime/v1/query?url=https%3a%2f%2fgithub.com%2fnotifications%2funsubscribe%2dauth%2fBC5GJFA2HENMXOIOVFXWB33X5KR3ZAVCNFSM6AAAAAAWKVGWLGVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TCNRUGEYTS&umid=348ab537-dc66-4575-9714-76943461be42&auth=f075895abcc5d3a93325341b3e31d07f62e0e5ed-50ea51b13f8c369fe704b9060999a3c6ff6e5f10>.
You are receiving this because you commented.Message ID: ***@***.***>


Email secured by Check Point

=

**Reply by StephanSchaller** [reply]
Hi Johanna,

this behaviour is not specific to PK.Sim but any PBPK tool.

In a PBPK Model, both the clearance and LogP has an influence on PK.

However, in the very early timepoints, it should not have much influence on a Cmax from an IV bolus injection (may be different for a infusion over a longer time period).

In any case, as both parameters have a strong influence on PK, I would recommend to use the parameter identification to estimate both.

Also, consider using in.vitro / the original physchem properties (for IVIVE) and not deviate too much from original input parameters.

Best,
Stephan

**Comment by JohannaZH2** [comment]
Hi Marjolein,

thank you very much for your answer. It's a pity that you currently have a break with PK Sim.
In fact, an expert from PK Sim answered my question very quickly.
We are now also trying to extend the application time, change the clearance and do a sensitivity analysis to figure out which parameters influence the half-life mostly.

If we find the solution for this problem, I will share it with you.

Stay in touch,
Best
Johanna

Von: MarjoleinvB ***@***.***>
Gesendet: Freitag, 29. September 2023 13:45
An: Open-Systems-Pharmacology/Forum ***@***.***>
Cc: Bouzid Johanna ***@***.***>; Comment ***@***.***>
Betreff: Re: [Open-Systems-Pharmacology/Forum] Terminal half life of midazolam in PBPK model (Discussion #1488)

Hi Johanna,

Thank you for contacting me! I am afraid I have to disappoint you. Due to lack of time, I temporarily stopped using and learning PK Sim and switched to Simcyp. (My research group has more experience with that platform).
However, I can advise you to put your problem on the Github forum with specifics on which model you use. I really quickly got a response by one of the PK Sim experts.

My question was focussed on that I want to use t1/2 as PK parameter to validate my PBPK models (together with Cmax and AUC). They only responded they don't advise t1/2, since it is calculated by the last 10% of data points of the terminal slope. With this in mind, the problem of the long half-life might be caused if the drug is not completely eliminated in the simulation or another factor why the last part of the curve is not correctly.

I am really interested in the solution. I hope to pick up my work in PK Sim in a couple of months and will ask a colleague who recently started to use PK Sim if he has problems with t1/2. If I figure out the problem, I am happy to discuss this with you. I hope you will find the solution.

Best,
Marjolein

Van: JohannaZH2 ***@***.***>
Verzonden: vrijdag 29 september 2023 12:06
Aan: Open-Systems-Pharmacology/Forum ***@***.***>
CC: Borselen, Marjolein van ***@***.***>; State change ***@***.***>
Onderwerp: Re: [Open-Systems-Pharmacology/Forum] Terminal half life of midazolam in PBPK model (Discussion #1488)


Hi Marjolein,
I read you question and would like to ask you whether you could find a way how to influence half-life value in PK Sim Simulations? I would be very happy to discuss this topic with you.
We try to get used to PK Sim and it is a common problem with our simulations, that half-life is too long as it should be.
We tried to chance logP and partition coefficient and cellular permeability, but it was not very successful.
Thank you and I am looking forward to hearing from you.
Best, Johanna

-
Reply to this email directly, view it on GitHub<https://github.com/Open-Systems-Pharmacology/Forum/discussions/1488#discussioncomment-7143040>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A6ZRZGZVYB4RG372CTOFV4TX42MSBANCNFSM6AAAAAAWKVGWLE>.
You are receiving this because you modified the open/close state.Message ID: ***@***.******@***.***>>

De informatie in dit bericht is uitsluitend bestemd voor de geadresseerde. Aan dit bericht en de bijlagen kunnen geen rechten worden ontleend. Heeft u deze e-mail onbedoeld ontvangen? Dan verzoeken wij u het te vernietigen en de afzender te informeren. Openbaar maken, kopi?ren en verspreiden van deze e-mail of informatie uit deze e-mail is alleen toegestaan met voorafgaande schriftelijke toestemming van de afzender. Het Radboudumc staat geregistreerd bij de Kamer van Koophandel in het handelsregister onder nummer 80262783.

The content of this message is intended solely for the addressee. No rights can be derived from this message or its attachments. If you are not the intended recipient, we kindly request you to delete the message and inform the sender. It is strictly prohibited to disclose, copy or distribute this email or the information inside it, without a written consent from the sender. Radboud university medical center is registered with the Dutch Chamber of Commerce trade register with number 80262783.


—
Reply to this email directly, view it on GitHub<https://smex-ctp.trendmicro.com:443/wis/clicktime/v1/query?url=https%3a%2f%2fgithub.com%2fOpen%2dSystems%2dPharmacology%2fForum%2fdiscussions%2f1488%23discussioncomment%2d7143746&umid=a49067b1-4b36-4893-abab-5f81a2589c8d&auth=796e4bc453b085770d7b9302191a1ec4575c461a-e53666ae9568f8d4f50cfdfd8659fa3a93947680>, or unsubscribe<https://smex-ctp.trendmicro.com:443/wis/clicktime/v1/query?url=https%3a%2f%2fgithub.com%2fnotifications%2funsubscribe%2dauth%2fBC5GJFE5D3LJ6ZCTYF3HCVLX42YC7ANCNFSM6AAAAAAWKVGWLE&umid=a49067b1-4b36-4893-abab-5f81a2589c8d&auth=796e4bc453b085770d7b9302191a1ec4575c461a-93bd59fdfb7d3da5c92f144fcbc48d6bcc6e5d16>.
You are receiving this because you commented.Message ID: ***@***.***>


Email secured by Check Point
